networking/external_dns_operator/external-dns-operator-release-notes.adoc

```diff
@@ -0,0 +1,56 @@
+// ExternalDNS Operator Release Notes
+:_mod-docs-content-type: ASSEMBLY
+[id="external-dns-operator-release-notes"]
+= ExternalDNS Operator release notes
+:context: external-dns-operator-release-notes
+include::_attributes/common-attributes.adoc[]
+
+toc::[]
+
+The ExternalDNS Operator deploys and manages DNS external to the node.
```

Comment:
I have 3 questions about this for your consideration. I'm not sure if any of these are relevant, but I'm just thinking out loud:

1. external DNS? I don't see any instances of "DNS external" in our docs: 
2. _to_ the node, or _on_ the node? 
3. "external DNS" <something>. Is it just external DNS? could it be like external DNS **infrastructure**? or maybe **resource**? sometimes in our docs its written as `ExternalDNS` with backticks and sometimes it isn't. I swear I'm not trying to open a can of worms here. It may be context dependent (i.e., when it is, when it isn't in backticks, and when it is/isn't capitalized). 

**BEYOND THE SCOPE OF THIS PR** I was looking up was ExternalDNS Operator was and landed on this page: https://docs.openshift.com/container-platform/4.10/networking/external_dns_operator/understanding-external-dns-operator.html 

There are some instances were ExternalDNS is in backticks, somewhere its written as ExternalDNS, and others where its written as "External DNS". On that page it's called the "External DNS Operator", but in the first sentence of the procedure, it says "You can deploy the ExternalDNS Operator on demand ". I'm not sure if this is your area but it might be worth fixing at some point, if you want. 

So it could be: 
```suggestion
The ExternalDNS Operator deploys and manages external DNS on the node.
```